src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage_WriteBatching.cs

```diff
@@ -1,245 +1,80 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
```

Comment:
this file's diff looks awful.  the 'old' part is irrelevant.  i recommend just looking at the 'new' side.